Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblocksview-tests', function (Y) {
+    var renderTest,
+        forwardActiveTest,
+        addBlockTest,
+        removeBlockTest,
+        VIEW_CONFIG = {container: '.container'},
+        SELECTOR_CONTENT = '.dashboard-content',
```

Comment:
should be `.ez-dashboard-content`, because of that the tests are failing it seems https://travis-ci.org/ezsystems/PlatformUIBundle/jobs/129451719#L383